docs/addons/instana-addon.md

```diff
@@ -0,0 +1,198 @@
+
+The IBM® Instana® [Addon](https://www.npmjs.com/package/%40instana/aws-eks-blueprint-addon) focuses on enhancing the user experience by reducing the complexity and time required to install and configure an Instana host agent on Amazon EKS cluster. Once you configure the addon for a Amazon EKS blueprint, it will be automatically provisioned during deployment.
+
+This Addon will use IBM Instana Kubernetes operator in the namespace ```instana-agent``` to install and manage the Instana agent. It also configures custom resource values to configure the operator.
+
+## Prerequisites
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [cdk](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+4. [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+5. Instana backend application - Use SaaS (eg [aws](https://aws.amazon.com/marketplace/pp/prodview-hnqy5e3t3fzda?sr=0-1&ref_=beagle&applicationId=AWSMPContessa)) or Install self-hosted Instana backend ([on-premises](https://www.ibm.com/docs/en/instana-observability/current?topic=installing-configuring-self-hosted-instana-backend-premises))
+
+## Installation
+
+To create a new project and install dependencies, follow these steps from Amazon EKS Blueprint [Quick Start](https://aws-quickstart.github.io/cdk-eks-blueprints/getting-started)
+
+Use following command to Install IBM Instana Addon:
+
+```shell
+npm i @instana/aws-eks-blueprint-addon
+```
+
+## Instana Agent Configuration
+Go to your Instana installation (Instana User Interface), click ... More > Agents > Installing Instana Agents and select 'Kubernetes' platform to get the Instana Agent Key, Instana Service Endpoint, Instana Service port. These steps are also described [here](https://www.ibm.com/docs/en/instana-observability/218?topic=instana-endpoints-keys) or in the screenshot below.
+
+![Instana Agent Configuration](../assets/images//instana-agent.png)
+
+To set the following environment variables from the CLI, use the corresponding values obtained from the Instana Service Endpoint and Port (as shown in the above screenshot), and the Instana Application Key (also shown in the above screenshot):
+
+- Set the value of **INSTANA_ENDPOINT_HOST_URL** to the Instana Service Endpoint.
+- Set the value of **INSTANA_ENDPOINT_HOST_PORT** to the Instana Service Port.
+- Set the value of **INSTANA_AGENT_KEY** to the Instana Application Key.
+
+You can choose the names for **AMAZON_EKS_CLUSTER_NAME** and **INSTANA_ZONE_NAME** based on your cluster's name, for example, "eks-blueprint".
+
+Set the value of the following environment variable and run it on CLI to set those variables.
+
+```
+export INSTANA_ZONE_NAME=
```

Comment:
This approach with env variables maybe fine for a pattern, but I really don't see a need to use ENV vars as opposed to literally setting them in the code or using standard CDK configuration mechanisms such as `cdk.context.json`. 

I am fine keeping it like that with a callout that it is one of ways. This approach won't work for pipelines support when Instance is deployed across a diverse enterprise landscape - which is the whole point of the addon.

Reply:
Done.